tools/scylla-nodetool.cc

```diff
@@ -107,6 +108,23 @@ const auto app_name = "nodetool";
 
 logging::logger nlog(format("scylla-{}", app_name));
 
+std::string doc_link(std::string_view url_tail) {
+    const auto product = scylla_product();
+    const auto version = scylla_version_short();
+
+    const auto prefix = product == "scylla-enterprise" ? "enterprise" : "opensource";
+
+    std::string branch = product == "scylla-enterprise" ? "enterprise" : "master";
+    nlog.info("doc_link(): product={}, version={}", product, version);
+    if (!version.ends_with("~dev")) {
+        std::vector<std::string> components;
+        boost::split(components, version, boost::algorithm::is_any_of("."));
+        branch = fmt::format("branch-{}.{}", components[0], components[1]);
+    }
+
+    return fmt::format("https://{}.docs.scylladb.com/{}/{}", prefix, branch, url_tail);
```

Comment:
Perhaps this should be moved to release.hh, since it's likely useful elsewhere. But we can do that when the time comes.

Reply:
Yes, I just found hardcoded links in the help output of scylla-sstable.

Reply:
Done in v3.